python/taichi/lang/impl.py

```diff
@@ -28,6 +28,21 @@ def expr_init(rhs):
             return Expr(taichi_lang_core.expr_var(Expr(rhs).ptr))
 
 
+@taichi_scope
+def expr_init_list(xs, expected):
+    if not isinstance(xs, (list, tuple)):
```

Comment:
Is it possible to support `ti.Vector`?

Reply:
Yes, as I mentioned above, let's fix this asap and open another issue on it.